translation/en/RedirectedResources/assets/abdata/adv/scenario/c30/14/94/translation.txt

```diff
@@ -1,7 +1,7 @@
-//「すぅ……すぅ……ん、んん……」=
-//（ううん、何だか下半身に違和感が……？）=
-//「…………」=
-//「って、うわぁ！　何して……んんっ……」=
-//「オチンポの味……ダメ……毎日咥えたくなっちゃう……」=
-//「ちょっ、少しは加減して……」=
-//「ふふっ、なにいってるの？　ここからが本番よ」=
+「すぅ……すぅ……ん、んん……」=(Hah... Hah... Nnnngg...)
+（ううん、何だか下半身に違和感が……？）=(...what's going on... my lower half feels weird...)
+「…………」=............
```

Comment:
Should be easy enough to clean up after, but this same thing is translated as a different number of `.` in a number of places